daprdocs/content/en/reference/components-reference/supported-middleware/middleware-invoke.md

```diff
@@ -0,0 +1,84 @@
+---
+type: docs
+title: "Invoke service"
```

Comment:
```suggestion
title: "Invoke service middleware"
```

It's best to craft titles that make sense with no other context, to best feed the search bar crawler